test/e2e/nodepool_kv_advanced_multinet_test.go

```diff
@@ -256,7 +256,7 @@ iptables -t nat -A PREROUTING -p tcp -d %[1]s -j DNAT --to-destination %[2]s
 	if err != nil {
 		return "", err
 	}
-	return e2eutil.RunCommandInPod(k.infra.Ctx(), infraClient, dnsmasqPod.Name, dnsmasqPod.Namespace, []string{"/bin/sh", "-c", command}, dnsmasqPod.Spec.Containers[0].Name)
+	return e2eutil.RunCommandInPod(k.infra.Ctx(), infraClient, dnsmasqPod.Name, dnsmasqPod.Namespace, []string{"/bin/sh", "-c", command}, dnsmasqPod.Spec.Containers[0].Name, 20*time.Minute)
```

Comment:
I have no idea if this takes a while or not, it's the only command we run that has any chance of doing so - figured I would be safe and not get bogged down in finding the perfect number in this PR.